packages/hub-threads-client/src/client.ts

```diff
@@ -67,13 +72,15 @@ Client.prototype.listThreads = async function (ctx?: Context): Promise<ListThrea
       .then((meta) => {
         client.listThreads(req, meta, (err: ServiceError | null, message: ListThreadsResponse | null) => {
           if (err) reject(err)
-          const msg = message?.toObject()
-          if (msg) {
-            msg.listList.forEach((thread) => {
-              thread.id = ThreadID.fromBytes(Buffer.from(thread.id as string, 'base64')).toString()
-            })
+          const lst = message?.getListList()
```

Comment:
These are lame, I wish we could configure that on our builds...

Reply:
once we hide the names from end users, they don't matter much